module/zfs/zvol.c

```diff
@@ -1478,9 +1478,9 @@ zvol_alloc(dev_t dev, const char *name)
 		goto out_queue;
 
 	/*
-	 * NOTE: Linux read-ahead is disabled in favor of zfs read-ahead.
+	 * NOTE: Linux read-ahead is limited to a page
 	 */
-	zv->zv_queue->backing_dev_info.ra_pages = 0;
+	zv->zv_queue->backing_dev_info.ra_pages = 1;
```

Comment:
This introduces a build failure on recent kernels which needs to be addressed.  It should also be squashed with the previous patch which entirely disabled read-ahead.

```
fs/zfs/zfs/zvol.c: In function ‘zvol_alloc’:
fs/zfs/zfs/zvol.c:1483:32: error: request for member ‘ra_pages’ in something not a structure or union
  zv->zv_queue->backing_dev_info.ra_pages = 1;
                                ^
```